slaves/linux-cross/arm-linux-gnueabihf.config

```diff
@@ -0,0 +1,577 @@
+#
+# Automatically generated file; DO NOT EDIT.
+# Crosstool-NG Configuration
+#
+# Generated with crosstool-ng-1.22.0's `ct-ng menuconfig` command using the default configuration
+# plus the following modifications
+#
+# Target options > Target Architecture = arm
+# Target options > Architecture level = armv7-a -- (*)
```

Comment:
> Eventually when we switch to rustbuild

(This is blocked on rust-lang/rust#32194 when the target is ARM :-/)

> Although now that I think about it I'm not sure it matters all that much. This largely only affects glibc, and we're not distributing glibc, so..

Yeah, in theory we always use the arm `gcc` with the right `-march` (provided by the build system) so this shouldn't really matter